src/pytest_bdd/scenario.py

```diff
@@ -197,14 +196,14 @@ def _execute_scenario(feature: Feature, scenario: Scenario, request: FixtureRequ
 
 def _get_scenario_decorator(
     feature: Feature, feature_name: str, templated_scenario: ScenarioTemplate, scenario_name: str
-) -> Callable[[Callable], Callable]:
+) -> Callable[[TCallable], TCallable]:
```

Comment:
ditto here, I think ParamSpec is really what we should use: https://docs.python.org/3.11/library/typing.html#typing.ParamSpec